src/GoMath.ts

```diff
@@ -37,6 +37,24 @@ export interface BoardState {
 
 type BoardTransform = (x: number, y: number) => { x: number; y: number };
 
+// OJE Sequence format is '.root.K10.Q1'  ...
+export function ojeSequenceToMoves(sequence: string): Array<JGOFMove> {
+    const plays = sequence.split(".");
+
+    if (plays.shift() !== "" || plays.shift() !== "root") {
+        throw new Error("Sequence passed to sequenceToMoves does not start with .root");
+    }
+
+    const moves = plays.map((play) => {
+        if (play === "pass") {
+            return { x: -1, y: -1 };
+        }
```

Comment:
Nit: `GoMath.decodeGTPCoordinate()` knows how to handle `pass`, so it's not necessary to check here.

https://github.com/online-go/goban/blob/3fd86d45a63170a1641dec96a2c6313c868ac6be/src/__tests__/GoMath.test.ts#L108-L111
https://github.com/online-go/goban/blob/3fd86d45a63170a1641dec96a2c6313c868ac6be/src/GoMath.ts#L221-L223